database/rrdset.c

```diff
@@ -860,6 +864,7 @@ RRDSET *rrdset_create_custom(
     rrdset_flag_clear(st, RRDSET_FLAG_DETAIL);
     rrdset_flag_clear(st, RRDSET_FLAG_DEBUG);
     rrdset_flag_clear(st, RRDSET_FLAG_OBSOLETE);
+    st->rrdhost->obsolete_charts_count--;
```

Comment:
This is on creation so the count will drop below zero (quickly prints 4294966968 on my test)

Reply:
Yeah, you are right. I've already saw that.